### ISSUE TITLE
Errors when using TS 1.8 string literal types

### ISSUE THREAD

**Issue by joeskeen** [palantir/tslint#966]
I'm using TS 1.8 beta, and when I try to use the new String Literal Types (https://github.com/Microsoft/TypeScript/pull/5185), I get errors:
```ts
type CardinalDirection = 'North' | 'East' | 'South' | 'West';
```

```
Failed to lint: 
... forbidden bitwise operation, 
... forbidden bitwise operation, 
... forbidden bitwise operation, 
... expected an assignment or function call.
```
I'm using the latest TSLint (2.5.1).

**Comment by JKillian** [comment]
@joeskeen Actually, you'll want `tslint@3.3.0` or `tslint@3.4.0-dev.1`. You shouldn't see the error anymore with either of those versions of TSLint :smiley:.

**Comment by joeskeen** [comment]
That works great!  
I knew I was missing something... thanks for the help! :)

**Comment by harangue** [comment]
I'm on `tslint@3.5.0` and I'm getting

> Type expected. The left-hand side of an arithmetic operation must be of type 'any', 'number', or an enum type.

```
type Valid = 'pending' | 'approved' | 'denied';
```

Do you know why?

**Comment by JKillian** [comment]
@harangue What version of TypeScript are you using?

**Comment by harangue** [comment]
Global and local are at 1.8.2.

**Comment by JKillian** [comment]
@harangue That error message is one generated by TypeScript (not TSLint). Perhaps the version of TypeScript your IDE is using is out of date?

**Comment by harangue** [comment]
@JKillian Sorry 'bout that - turns out it was VSCode's fault. Apparently it has its own separate TypeScript install separate from global and local installations and I mistakenly took it for a tslint issue. Bleh.

**Comment by nikkwong** [comment]
Unfortunately I'm still getting this issue on ```tslint 3.7.4```, though I'm running it through ```gulp-tslint@3.6.0``` which should defer to the local ```tslint```. How can I go about troubleshooting this? I also have the latest typescript locally and globally. 

Thanks.
